build.gradle

```diff
@@ -1,56 +1,58 @@
 import java.nio.file.Files
+import java.nio.file.Paths
 import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
 
 apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'maven-publish'
 
-def CLOUD_PUBSUB_VERSION = '1.37.1'
+group "org.logstash.inputs"
+version Files.readAllLines(Paths.get("version")).first()
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    compile "com.google.cloud:google-cloud-pubsub:${CLOUD_PUBSUB_VERSION}"
+    implementation "com.google.cloud:google-cloud-pubsub:1.115.1"
```

Comment:
Any reason why this version?

Reply:
Because Logstash 8.3 provides google guava [31.0.1](https://github.com/elastic/logstash/blob/8.3/logstash-core/build.gradle#L182) which was published on [19 Mar 2021](https://search.maven.org/artifact/com.google.guava/guava) and this is a version of the client that depends transitively on the same:
```
+--- com.google.cloud:google-cloud-pubsub:1.115.1
|    +--- io.grpc:grpc-api:1.43.2
|    |    +--- io.grpc:grpc-context:1.43.2
|    |    +--- com.google.code.findbugs:jsr305:3.0.2
|    |    +--- com.google.guava:guava:30.1.1-android
```

Reply:
we can try to bring the latest (`1.120.13`) and check if everything is ok